app/models/manageiq/providers/redhat/inventory/persister/target_collection.rb

```diff
@@ -5,17 +5,6 @@ def initialize_inventory_collections
     initialize_infra_inventory_collections
 
     @collection_group = nil
-
-    add_collection(infra, :vm_and_template_ems_custom_fields) do |builder|
```

Comment:
@Ladas This definition overwrites https://github.com/ManageIQ/manageiq-providers-ovirt/pull/268/files#diff-d512eba5610c33fabd8eefbdd6f811e2R84 (in `@collections`) so it can be merged together.
Rest of properties are defined in core, so they are not needed in both definitions.